server/lib/transforms/insert-ad.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const getMiddleParagraph = require('./get-middle-paragraph');
+const adMarkup = () => `<div class="ad-container"><amp-ad width="300" height="250" type="doubleclick"></amp-ad></div>`;
+
+module.exports = function insertAd($, options) {
+	getMiddleParagraph($).after(adMarkup(options.targeting));
```

Comment:
Just had a thought about this: now ads and light signup have the same positioning but ads runs first, light signup is likely to be placed lower down the page. I'm not sure if that's going to be acceptable, I'll check with Martin, but we'll probably have to tweak this.